RallyHereIntegration/Source/RallyHereEditor/Public/RH_DevIntegrationSettings.h

```diff
@@ -0,0 +1,120 @@
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "UObject/Object.h"
+
+#include "Engine/DeveloperSettings.h"
+
+#include "RH_DevIntegrationSettings.generated.h"
+
+/** @defgroup IntegrationSettings RallyHere Integration Settings
+ *  @{
+ */
+
+/**
+ * @brief Settings for Sandbox Configuration.  If set, these override the defaults when the SandboxId matches the specified sandbox id
+ */
+USTRUCT()
+struct FRH_DevSandboxConfiguration
+{
+	GENERATED_BODY()
+	/** @brief Sandbox Name, for use with client side sandbox lookups and overrides. */
+	UPROPERTY(EditAnywhere, Config, Category = "Connectivity")
+	FString SandboxId;
+
+	/** @brief RallyHere BaseURL. Determines the base URL to use when connecting to the RallyHere API */
+	UPROPERTY(EditAnywhere, Config, Category = "Connectivity")
+	FString BaseUrl;
+
+	/** @brief RallyHere BaseURL. Determines the base URL to use when Authing for the RallyHereAPI */
+	UPROPERTY(EditAnywhere, Config, Category = "Connectivity")
+	FString AuthUrl;
+
+	/** @brief RallyHere Client ID. Determines the available permissions for RallyHere APIs */
+	UPROPERTY(EditAnywhere, Config, Category = "Connectivity")
+	FString ClientId;
+
+	/** @brief RallyHere Client secret. Must be associated with the defined client ID */
+	UPROPERTY(EditAnywhere, Config, Category = "Connectivity")
+	FString ClientSecret;
+
+	/** @brief Sandbox Guid for the environment to be used for updating dev portal data */
+	UPROPERTY(EditAnywhere, Config, Category = "Connectivity")
+	FString SandboxGuid;
+};
+
+/**
+ * @brief Main settings for the Integration.
+ */
+UCLASS(Config = RallyHereIntegration, DefaultConfig, meta = (DisplayName = "Rally Here Integration Settings"))
```

Comment:
Putting this in the main config file as opposed to an "Editor" config file could be problematic.  Would more correctly belong in DefaultEditorUserSettings if we are not providing defaults.  If we need to have a RallyHereIntegrationEditor set of inis that is somewhat annoying (more inis) but more correct.

Reply:
Do we need to have any default configuration for it from the plugin level, or is it all project / user level?

Reply:
nothing default, all at project level.

Reply:
Lets put it in the editor user settings then.  There is a default file for that at project level for teams to use to set their connection information, and it gets some automatic handling around builds and the editor.

Reply:
I would guess PerProject would be correct here, as different projects would have different settings